hepcrawl/spiders/aps_spider.py

```diff
@@ -199,3 +213,6 @@ def _get_authors_and_collab(self, article):
 
     def _file_name_from_url(self, url):
         return "{}.xml".format(url[url.rfind('/') + 1:])
+
+    def make_file_fingerprint(self, set_):
+        return u'metadataPrefix={}&set={}'.format(self.format, set_)
```

Comment:
where is the `format` coming from? do you really need that part? it seems OAI-PMH specific. Are you sure this really works?